Makefile

```diff
@@ -13,6 +13,7 @@ DESTDIR =
 PREFIX = /usr/local
 MANDIR = $(PREFIX)/share/man
 MAN5DIR = $(MANDIR)/man5
+DATADIR = $(PREFIX)/share/eclass-manpages
```

Comment:
Script is not data, it's code, so go for `/usr/lib` aka LIBEXECDIR.

Reply:
It's not a binary, but architecture-independent data, therefore goes to `/usr/share`? That's also where most packages (most notably, `sys-apps/gawk` itself) install these scripts:
`$ find /usr/share -name '*.awk' | wc -l`
`52`
`$ find /usr/lib{,exec} -name '*.awk' | wc -l`
`1`